TAsk4(attempt4)/TAsk4(attempt4)/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TAsk4_attempt4_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double a = Koren(2, 3);
+            double b = Koren(7, 8);
+            double c = Koren(1, 4);
+            double result = a + b + c;
```

Comment:
Можно было проще без лишних локальных переменных
```suggestion
            double result = Koren(2, 3) + Koren(7, 8) + Koren(1, 4);
```